.github/workflows/ci.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       matrix:
         # Add '3.10-dev' once frozendict issue is resolved
         # See https://github.com/starkware-libs/cairo-lang/issues/38
-        python_version: [ '3.8', '3.9' ]
+        python_version: [ '3.9' ]
```

Comment:
what was the need for this?

Reply:
I guess bad assumption on my part? The `cairo-lang` v0.10 release stated they changed from python3.7 to 3.9 ([release](https://github.com/starkware-libs/cairo-lang/releases/tag/v0.10.0)) and their documentation changed their environment setup from using python3.7 to 3.9 (see [quickstart](https://www.cairo-lang.org/docs/quickstart.html)). python3.8 still seems to work (at least with Nile). I suppose we should keep it unless we find it's broken. What do you think?

Reply:
if cairo-lang works, let's maintain it